scripts/release-master/publish-to-github.js

```diff
@@ -0,0 +1,89 @@
+const readline = require('readline');
+const fs = require('fs');
+const Octokit = require('@octokit/rest');
+
+module.exports = async function publishToGitHub(version, { GITHUB_TOKEN }) {
+  const octokit = new Octokit({ auth: `token ${GITHUB_TOKEN}` });
+
+  console.log('Extract section from CHANGELOG.md');
+  const changeLog = await getChangeLogSection(version);
+  const body = [
+    '```',
+    `https://cdn.webrtc.ecl.ntt.com/skyway-${version}.js`,
+    '```',
+    ...changeLog,
+  ].join('\n');
+  console.log('');
+
+  console.log('Create new release');
+  const { data: { upload_url } } = await octokit.repos.createRelease({
+    owner: 'skyway',
+    repo: 'skyway-js-sdk',
+    tag_name: `v${version}`,
+    target_commitish: 'master',
+    name: `v${version}`,
+    body,
+  });
+  console.log('');
+
+  console.log('Upload release assets');
+  const sdkDev = fs.readFileSync('./dist/skyway.js');
+  const sdkMin = fs.readFileSync('./dist/skyway.min.js');
+
+  await Promise.all([
+    octokit.repos.uploadReleaseAsset({
+      headers: {
+        'content-length': sdkDev.length,
+        'content-type': 'application/javascript',
+      },
+      url: upload_url,
+      name: 'skyway.js',
+      file: sdkDev,
+    }),
+    octokit.repos.uploadReleaseAsset({
+      headers: {
+        'content-length': sdkMin.length,
+        'content-type': 'application/javascript',
+      },
+      url: upload_url,
+      name: 'skyway.min.js',
+      file: sdkMin,
+    }),
+  ]);
+};
+
+function getChangeLogSection(version) {
+  return new Promise(resolve => {
+    const rl = readline.createInterface({
+      input: fs.createReadStream('./CHANGELOG.md'),
+      crlfDelay: Infinity,
+    });
+
+    const lines = [];
+    let isTargetStart = false;
+    let isTargetEnd = false;
+    rl.on('line', line => {
+      const isVersionLine = line.startsWith('## ');
+      const isVersionFound = line.includes(`v${version}`);
+
+      if (isTargetEnd) {
+        // 4. next section found, quit
+        resolve(lines);
+      } else if (isTargetStart === false) {
+        // 1. target section found
+        isTargetStart = isVersionLine && isVersionFound;
+        // just set start flag
+      } else {
+        // 2. check end flag
+        isTargetEnd = isVersionLine && !isVersionFound;
```

Comment:
I misread that the `isVersionFound` bool never becomes true.
Any line which starts with '## ' includes version number at least for now.
However, this condition means the line starts with '## ' and not includes the DESIRED release version number.
It's not obvious from the condition variable's names because the article is omitted.